### PR TITLE
Added lib generated files to send telemetry data to VAC API

### PR DESCRIPTION
**Pull request by snootan**
Added all generated files for go-client to send telemetry data of cluster to VAC API.

### CODE REVIEW

File: src/api/v1alpha1/workload.go [hunk 1]
```diff
@@ -39,6 +40,8 @@ var AllWorkloads = []string{
 type Workload struct {
 	// For pure Pod, no object reference.
 	corev1.ObjectReference `json:"metadata"`
+	// Replicas of this workload.
+	Replicas appsv1.ReplicaSet `json:"replicas"`
```
Comment (mpermar) [inline src/api/v1alpha1/workload.go:44]: I see some of these replica set related changes. Why do we need these changes? Is this some gap in Project Narrows that we found?

File: src/config/crd/bases/goharbor.goharbor.io_assessmentreports.yaml [hunk 2]
```diff
@@ -128,6 +128,23 @@ spec:
                       elasticSearchUser:
                         description: ElasticSearch username for the client
                         type: string
+                      governor:
```
Comment (mpermar) [inline src/config/crd/bases/goharbor.goharbor.io_assessmentreports.yaml:131]: Why do we need the governor configuration on each assessment report? That looks strange as there are going to be many of those.

File: src/pkg/inspection/controller.go [hunk 3]
```diff
@@ -331,6 +332,20 @@ func (c *controller) Run(ctx context.Context, policy *v1alpha1.InspectionPolicy)
...
 
+	// Read config from InspectionPolicy, send assessment reports to Governor api if governor enabled.
+	if policy.Spec.Inspection.Assessment.Governor.Enabled {
+		governorConfig := policy.Spec.Inspection.Assessment.Governor
+		exporter := governor.GovernorExporter{
+			Report:    report,
+			ClusterID: governorConfig.ClusterID,
+			ApiURL:    governorConfig.URL,
+			ApiToken:  governorConfig.APIToken,
+		}
```
Comment (mpermar) [inline src/pkg/inspection/controller.go:343]: This block can probably out of the loop. The inspection policy data is not going to change.

**Comment by snootan** [comment]
Closing this PR as this was just testing purpose. @mpermar, will check your comment in [this PR](https://github.com/vmware-tanzu/cloud-native-security-inspector/pull/170)

**Pull request closed without merging**